eth/tracers/api_test.go

```diff
@@ -187,6 +188,90 @@ func (b *testBackend) StateAtTransaction(ctx context.Context, block *types.Block
 	return nil, vm.BlockContext{}, nil, nil, fmt.Errorf("transaction index %d out of range for block %#x", txIndex, block.Hash())
 }
 
+type stateTracer struct {
+	Balance map[common.Address]*hexutil.Big
+	Nonce   map[common.Address]hexutil.Uint64
+	Storage map[common.Address]map[common.Hash]common.Hash
+}
+
+func newStateTracer(ctx *Context, cfg json.RawMessage, chainCfg *params.ChainConfig) (*Tracer, error) {
+	t := &stateTracer{Balance: make(map[common.Address]*hexutil.Big), Nonce: make(map[common.Address]hexutil.Uint64), Storage: make(map[common.Address]map[common.Hash]common.Hash)}
```

Comment:
nitpick, can you change the initialization style a bit to 

```go
    t := &stateTracer{
        Balance: make(map[common.Address]*hexutil.Big), 
        Nonce:   make(map[common.Address]hexutil.Uint64), 
        Storage: make(map[common.Address]map[common.Hash]common.Hash),
    }
```

it's easier to read

Reply:
Yep agreed 👍